cwltool/process.py

```diff
@@ -407,13 +407,20 @@ def _realpath(
         if cast(str, ob["location"]).startswith("/"):
             ob["location"] = os.path.realpath(cast(str, ob["location"]))
 
+
     outfiles = list(_collectDirEntries(outputObj))
     visit_class(outfiles, ("File", "Directory"), _realpath)
     pm = path_mapper(outfiles, "", destination_path, separateDirs=False)
     stage_files(pm, stage_func=_relocate, symlink=False, fix_conflicts=True)
 
     def _check_adjust(a_file: CWLObjectType) -> CWLObjectType:
-        a_file["location"] = file_uri(pm.mapper(cast(str, a_file["location"]))[1])
+        location = cast(str, a_file["location"])
+        _logger.debug("KM:: for file {} location is {}".format( a_file, location ))
```

Comment:
When you're ready for review, please remove this logging

Reply:
The logging line is removed